pkg/costmodel/allocation_helpers.go

```diff
@@ -1707,12 +1707,24 @@ func (cm *CostModel) applyNodesToPod(podMap map[podKey]*pod, nodeMap map[nodeKey
 
 // getCustomNodePricing converts the CostModel's configured custom pricing
 // values into a nodePricing instance.
-func (cm *CostModel) getCustomNodePricing(spot bool, providerID string) *nodePricing {
+func (cm *CostModel) getCustomNodePricing(spot bool, providerID, nodeName string) *nodePricing {
 	customPricingConfig, err := cm.Provider.GetConfig()
 	if err != nil {
 		return nil
 	}
 
+	if nodeName != "" {
```

Comment:
@mattray Maybe i misunderstand something, but our pricing model doesn't fit to the default `customPricingConfig.CPU` model, so we need to find node price, because our `CostPerCPUHr` is different by node types. WDYT?

Reply:
I found a way to avoid this change.